draft-ietf-quic-multipath.md

```diff
@@ -319,6 +319,13 @@ to migrate to that path.  Instead, servers SHOULD consider new paths
 over which non-probing packets have been received as available
 for transmission.
 
+As specified in {{Section 9.3 of QUIC-TRANSPORT}}, the server SHOULD send a new
+address validation token to a client following the successful validation of a
+new client address. In situations where multiple paths are activated, the
+client may be recipient of several tokens, each tied to a different address.
+When considering using a token for subsequent connections, the client ought to
+carefully select the token to use, due to the inherent ambiguity associated
+with determining the exact path to which a token is bound.
```

Comment:
```suggestion
client might have difficulty in determining which token validates which address.
To alleviate this difficulty, servers can issue a token that could be used for
validating any of the validated addresses.
```

Reply:
A token that can be used for validating multiple address works for me. Looking at the discussion #248, it seems that there was still a different opinion. Do you have a second opinion on this @huitema ?

Reply:
I think the "token validates multiple addresses" is probably the best approach. The hidden assumptions are:

1- Most clients do not actually know their source address before contacting as server, because of mobility and NAT
2- Clients are like to just keep the last token provided by the server and reuse it.

If a server sends tokens that validate the list of addresses seen so far and the client keeps the latest token for that server, then yes it works.

Reply:
This is fine for me but there is still some ambiguity about which address the token applies to. Do we need any normative language here, like the client SHOULD only use the latest token ? I guess that would be divergence RFC9000.